task.js

```diff
@@ -0,0 +1,193 @@
+// you can see an example of interaction with the order at the end of the file
+
+class Item {
+    constructor(price, calories) {
+        this.price = price;
+        this.calories = calories;
+    }
+
+    calculatePrice() {
```

Comment:
instead of putting `calculatePrice` and `calculateCalories` I would suggest you to put in abstract class only methods `getPrice` and `getCalories`.